Sources/UtiliKit/Container/ContainerViewController.swift

```diff
@@ -162,7 +169,23 @@ private extension ContainerViewController {
             visibleController = source
         }
         
+        switch postTransitionBehavior {
+        case .removeAllNonVisibleChildren:
+            removeAllNonVisibleChildren(except: [])
+        case .removeAllNonVisibleChildrenExcept(let identifiers):
+            removeAllNonVisibleChildren(except: identifiers)
+        case .none: break
```

Comment:
I don't think UtiliKit is using SwiftLint but I know you could get the following using it:
Switch Case on Newline Violation: Cases inside a switch should always be on a newline (switch_case_on_newline)
Can you move the break to a new line.
```suggestion
        case .none:
            break
```

Reply:
Done